packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -336,6 +340,15 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
   const defaultOnRenderVideoTile = useCallback(
     (participant: VideoGalleryRemoteParticipant, isVideoParticipant?: boolean) => {
       const remoteVideoStream = participant.videoStream;
+      // Async call to retrieve custom data model information to replace display name in remote tiles with camera on.
+      if (onFetchAvatarPersonaData) {
+        (async () => {
+          const newPaticipantData = await onFetchAvatarPersonaData(participant.userId);
+          participant.displayName = newPaticipantData.text;
```

Comment:
AvatarPersonaData has many fields, but here we only overwrite the displayName prop - is there a plan to support the other props in this interface?

Reply:
All of the other properties of the avatarPersonaData are only related to the avatar that is present when the camera is off and in the participant list in its many forms. the key issue here unfortunately is that the display name that the video gallery is placing on each tile isn't getting the new `text` or name of the participant that shows in the participant list.